PSSwagger/PSSwagger.psm1

```diff
@@ -468,6 +479,19 @@ function New-PSSwaggerModule {
         }
     }
 
+    if (-not $Formatter -and $PowerShellCodeGen['Formatter']) {
+        $Formatter = $PowerShellCodeGen['Formatter']
+    }
+
+    if ($Formatter) {
+        if ($Formatter -eq 'PSScriptAnalyzer') {
+            if (-not (Get-Module PSScriptAnalyzer)) {
```

Comment:
>Get-Module PSScriptAnalyzer [](start = 22, length = 27)

Consider adding -ListAvailable switch here.
PSScriptAnalyzer module can be installed but not required to be imported by the user.